bin/ezmlm-send.c

```diff
@@ -335,21 +339,39 @@ static void rewrite_from()
 
   if (flagrewritefrom) {
     concatHDR(from.s,from.len,&lines);
-    author_name(&author,lines.s,lines.len);
+    author_name(&author,lines.s,lines.len,1);
     stralloc_0(&author);
     stralloc_copy(&dummy,&outlocal);
     stralloc_0(&dummy);
 
-    stralloc_copyb(&line,"From: \"",7);
+    stralloc_copyb(&line,"From: ",6);
     stralloc_cats(&line,MSG2(AUTHOR_VIA_LIST,author.s,dummy.s));
-    stralloc_catb(&line,"\" <",3);
+    stralloc_catb(&line," <",2);
     stralloc_catb(&line,outlocal.s,outlocal.len);
     stralloc_catb(&line,"@",1);
     stralloc_catb(&line,outhost.s,outhost.len);
     stralloc_catb(&line,">\n",2);
-    stralloc_cats(&line,flagreplytolist ? "Cc:" : "Reply-To:");
+    if (flagreplytolist) {
+      if (cc.s) {
+        --cc.len;	/* remove '\n' */
+        stralloc_catb(&cc,",\n",2);
+      }
+      stralloc_catb(&cc,from.s,from.len);
```

Comment:
This assumes `from` starts with whitespace (otherwise the appended value will not be considered part of the Cc header, but as the start of a new header), which is not guaranteed. Probably just insert a ` `?

Reply:
I think I remember reading in RFC 5322 (or potentially one of the earlier RFCs defining the email message format) that whitespace is actually required between the `:` and header value.  So there should be some whitespace there from when the `From` header value was saved (lines 656-657):
```c
        } else if (case_startb(line.s,line.len,"From:")) {
	  stralloc_copyb(&from,line.s+5,line.len-5);
```

Though I recognize ezmlm-send could be fed a non-compliant message—so it's probably not a bad idea to ensure there's leading whitespace for the continuation line, as you suggest, especially if ezmlm-send isn't otherwise assuming there is whitespace separating the header names and values.

Reply:
it's definitely common but I didn't find it in https://www.rfc-editor.org/rfc/rfc5322#section-3, and the unittests didn't complain when I created a `From:` that was not followed by whitespace.

Reply:
Addressed by commit [17d9159](https://github.com/bruceg/ezmlm-idx/pull/20/commits/17d91594efebf6bff5abc35680257ab359ecda6e).